src/index.js

```diff
@@ -1,52 +1,74 @@
 #!/usr/bin/env node
-// @flow
 
-const USAGE = '[env]'
+const USAGE = '[action] [env]'
 
-const fs = require('fs')
 const program = require('commander')
-
 const { fatal } = require('./util')
-const { deployNodeApp } = require('./deployNodeApp')
-// eslint-disable-next-line
-const dnaPackageJson = require(__dirname + '/../package.json')
-
-let packageJson
-try {
-  packageJson = JSON.parse(fs.readFileSync('package.json'))
-} catch (err) {
-  fatal('This doesn\'t appear to be a Node.js application. You may need to \'npm init\'.')
-}
-if (typeof packageJson.name !== 'string') {
-  fatal('Please add a name to your package.json and re-run')
-}
+const deployNodeApp = require('./deployNodeApp')
+const dnaPackageJson = require(__dirname + '/../package.json') // eslint-disable-line
+
+const languages = [
+  require('./languages/nginx'),
+  require('./languages/nodejs'),
+  require('./languages/php'),
+  require('./languages/python')
+]
+
+let env
+let action
 
 program
-  .arguments('[env]')
+  .name('deploy-to-kube')
```

Comment:
Isn't this supposed to be `deploy-node-app`?

Reply:
Yes it is - We're potentially going to rename this project `deploy-to-kube` due to the fact that this new version should happily support other languages (as long as we can somehow guess their dependencies in a regular way). But we've decided to punt that for now, as Node.js is the only first-class citizen that we've been heavily testing for now. Reverted - thank you for the review!!!